gallery/src/styles/NxTile/NxTilesExamples.tsx

```diff
@@ -152,6 +154,14 @@ const NxTilesExamples = () =>
       An example of a tile with an <NxCode>nx-grid</NxCode> as the content, demonstrating that the spacing comes
       out as expected.
     </GalleryExampleTile>
+
+    <GalleryExampleTile title="Simple Convenience Components Example"
```

Comment:
Needs `defaultCheckeredBackground={true}`.  This will make the visual test snapshot clearer too

Reply:
> Needs `defaultCheckeredBackground={true}`. This will make the visual test snapshot clearer too

I added that here a0e0084e4b69ed42779a44b25097c40538e19713
For some reason the snapshots didn't update or the checkered board is not showing in the visual tests. taking a look at that now

Reply:
> Needs `defaultCheckeredBackground={true}`. This will make the visual test snapshot clearer too

Updated snapshots here: 99e69dadf1767a2ef9d30cf8d02712388d4ee26f